src/Shouldly/Internals/Assertions/StringShouldBeAssertion.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Shouldly.DifferenceHighlighting2;
+using System;
+
+namespace Shouldly.Internals.Assertions
+{
+    internal class StringShouldBeAssertion : IAssertion
```

Comment:
Just looking into this a bit more with the xunit changes. Why not use the message generator patterns?